src/components/CreateAnalysis/CreateAnalysisDetail.tsx

```diff
@@ -1,22 +1,29 @@
-import React, { Dispatch, SetStateAction, useContext, useEffect } from "react";
+import React, { Dispatch, SetStateAction, useContext, useEffect, useState } from "react";
 import avator from '../../assets/images/avator.png';
 import { CreateAnalysisTypes } from "../../context/actions/types";
 import { AppContext } from "../../context/context";
 import RightArrowButton from "../../pages/CreateAnalysisPage/RightArrowButton";
 import CreateAnalysisService, { StudyInstance } from "../../services/CreateAnalysisService";
+import ModelSelection from "./ModelSelection";
 import SelectedStudyDetail from "./SelectedStudyDetail";
 import SelectionStudy from "./SelectionStudy";
+import plugins from '../../api/app.config'
 
 
 interface CreateAnalysisDetailProps {
   setIsExpanded: Dispatch<SetStateAction<boolean>>,
-  submitAnalysis: () => void
+  submitAnalysis: (XrayModel: string | void, CTModel: string | void) => (() => void)
```

Comment:
Did you mean that `XrayModel` and `CTModel` can be `undefined`? In that case, I would suggest 
```suggestion
  submitAnalysis: (XrayModel?: string, CTModel?: string) => (() => void)
```